opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -1167,16 +1169,37 @@ def get_tracer(
             logger.error("get_tracer called with missing module name.")
         if instrumenting_library_version is None:
             instrumenting_library_version = ""
+
+        kwargs = {
+            "message": (
+                "Call to deprecated method __init__. \(You should use "
+                "InstrumentationScope\) -- Deprecated since version 1.11.1."
+            ),
+            "category": DeprecationWarning,
+            "module": "opentelemetry.sdk.trace",
+        }
+
+        # FIXME: Remove when 3.7 is no longer supported.
+        # For some reason the lineno argument in 3.7 does not seem to work.
+        # This is still safe enough because we also filter by message and
+        # module.
+        if version_info.minor > 7:
+            kwargs["lineno"]: 1191
+
+        filterwarnings("ignore", **kwargs)
```

Comment:
Do you need to reset this at the end of the function?

Reply:
No, resetting the filters causes the issue we are trying to solve here, this is why we use specific filter arguments like message, module and line number. The problem is that the function that resets the filters resets everything, there is no way to remove only one filter which is what we need.